CMakeLists.txt

```diff
@@ -245,9 +245,9 @@ target_include_directories(${PROJECT_NAME} PUBLIC
 )
 
 # TODO(damonkohler): Create a testing library.
```

Comment:
Fine, you can remove the TODO and these lines.

Reply:
:+1: second commit amended accordingly